README.md

```diff
@@ -107,6 +109,10 @@ Additionally, an optional `HTTPS_PROXY` variable can be set to enable a proxy fo
 
 - `NEW_RELIC_APPLICATION_NAME` (optional) The name of the PHP application to instrument. This name will be listed under New Relic's `APM & Services`. If omitted, defaults to `PHP Application`.
 
+`apache`:
+
+- `NR_CLI_STUB_STATUS_URL` (optional) The URL to check the Apache web server status. This is used to ensure that an Apache web server is running on the host and is in a healthy state before attempting the installation of the Apache On-Host Integration.
```

Comment:
Since so far, all exposed env vars are consistently named as *NEW_RELIC_...*, should we consider renaming this variable in the recipe itself firstly accordingly? cc: @kidk

Reply:
What do we think about namespacing some of these CLI vars to the integration? Starting this practice would help reduce the potential of collisions for people with complicated sets of targets.

`NEW_RELIC_APACHE_STATUS_URL`?

Reply:
I'm not sure this is warranted, we may have colliding names down the road, when we do we can disambiguate at that time. In the meantime, this matches the naming in the doc, with the exception of our newrelic prefix (which is consistent/are trying to make consistent).